lib/github.js

```diff
@@ -147,8 +147,8 @@ github.prototype = {
 							return false;
 
 
-						if(b.state == "APPROVED")
-							return true;
+                        if(b.state == "APPROVED" || b.state == "CHANGES_REQUESTED")
+                            return true;
 
 						if(b.submitted_at < pullRequest.updated_at)
```

Comment:
@leizmonk this won't work .. If you requested changes you shouldn't be notified unless your changes were addressed by the author .. checking this is complicated and so line 153 makes sure that you are notified if the pull request has changed after you submitted a review and does not notify you otherwise ...

If we wish to enhance it we need to think of a way to know that these changes address the reviewer requested changes and not those of another reviewer